sections/introduction/project-idea.tex

```diff
@@ -0,0 +1,24 @@
+\section{Project idea}
+The idea for this project is to create a game using augmented reality (AR). 
+Two teams will compete againts each other to score the most goals using a ball. 
+Each player will be equipped with a Google Cardboard, and these will display the playing field from a top-down 2D view. 
+To achieve this, each player's position needs to be tracked as well as where the ball located on the field.
+In the top down view each player needs to see the positions of the other players and the ball.
+They also need to see their position on the playing field and where the goalposts are.
+The players should be able to set an amount of goals they need to score to win before begining the game. 
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.6\linewidth]{Spil.PNG}
+    \caption{A game.}
+    \label{fig:game_illustration}
+\end{figure}
+In \autoref{fig:game_illustration} an illustration of the playing field for the game is shown. There are goalposts in each end of the field and the teams score goals by getting the ball between the goalposts. There could also be another version of the game where instead of goalposts, there are goal zones where the teams need to bring the ball to.
```

Comment:
```suggestion
In \autoref{fig:game_illustration}, an illustration of the playing field for the game is shown.
There are goalposts in each end of the field, and the teams score goals by getting the ball between the goalposts.
There could also be another version of the game where, instead of goalposts, there are goal zones into which the teams need to bring the ball.
These zones could even change locations as the game progressed.
```